src/rex.ctl/src/rex/ctl/core.py

```diff
@@ -576,16 +577,13 @@ def prompt(msg):
 
 
 env = Environment()
-env.add(shell=Environment(name="Rex",
+env.add(shell=Environment(name="rex",
                           description="""Command-line administration utility"""
                                       """ for the RexDB platform""",
                           local_package='rex.local',
                           entry_point='rex.ctl',
-                          config_name='rex.yaml',
-                          config_dirs=['/etc',
-                                       os.path.join(sys.prefix, '/etc'),
-                                       os.path.expanduser('~/.rex'),
-                                       os.path.abspath('.')]),
+                          config_dirs=[os.path.abspath('.'), sys.prefix]),
```

Comment:
Do we actually need to look for configs in `.`, is it used somewhere? If not (or it's used just in a few places) I think it's better to leave just `sys.prefix` and convert those places to be explicit about their behaviour with `--config ./APP.yaml`.